storybook/cypress/e2e/modal.cy.ts

```diff
@@ -22,9 +22,10 @@ describe("Modal", () => {
       .should("have.attr", "role", "document")
       .find(".dso-close span")
       .should("have.class", "sr-only")
-      .and("have.text", "Sluiten");
+      .and("have.text", "Sluiten")
+      .should("exist");
```

Comment:
Is deze test nodig? Er is al getest of het gevonden element een class `sr-only` heeft, ik denk dat we er veilig vanuit kunnen gaan dat het element dan bestaat 😄

Reply:
Heb de exist() weggehaald.

Echter met en zonder zie ik momenteel geen modal in de screenshot, doch slechts een wit vlak:

![Modal -- should be accessible snap](https://github.com/dso-toolkit/dso-toolkit/assets/4646174/f2f0ce25-4f87-45dd-bb3a-d7adcdaf760d)

Reply:
.. en dus wait(100) incl. een comment toegevoegd

Reply:
Ok, gaan we naar kijken. Deze resolve ik, op naar de volgende thread

Reply:
de wait is nu gelijk aan de modal animation-duration of 200ms